File: soliderAndBananas.cpp
#include<iostream>
using namespace std;

int main(){

	int k,n,w;
	cin >> k >> n>> w;

	int money = 0;

	for(int i=1;i<=w;i++){
		money += k*i;
	}

	if(money > n){
		cout<< money - n;
	}else{
		cout<< 0;
	}
	
	return 0;
}

File: soliderAndBananas.js
var soliderAndBananas =  (k,n,w)=>{
	var money = 0;

	for(var i=1;i<=w;i++){
		money += k*i;
	}

	if(money > n){
		return money - n;
	}else{
		return 0;
	}
}

console.log(soliderAndBananas(3,17,4))
 